internal/model/serverservice.go

```diff
@@ -25,6 +25,9 @@ const (
 	// errors that occurred when connecting/collecting inventory from the bmc are stored here.
 	ServerBMCErrorsAttributeNS = ServerServiceNSPrefix + ".server_bmc_errors"
 
+	// BIOS configuration is stored in this namespace.
+	ServerBIOSConfigNS = "net.platformequinix.bios"
```

Comment:
Since this is Equinix specific, and that we're going to be adding functionality to collect the same data out of band, I suggest
 - We check if anyone really depends on the NS at the moment
         - if its not critical to keep, use the `ServerServiceNSPrefix` value instead
         - if it has to stay then use `ServerServiceNSPrefix` as the default and set this through a configuration parameter

- Add a function like the one linked below, to return an NS based on the `appKind` value, so that way the inband and OOB data is kept separated https://github.com/metal-toolbox/alloy/blob/8e422223aad117480ae80c55a05184640c710eca/internal/model/serverservice.go#L42

Reply:
Fixed in [a885fbb](https://github.com/metal-toolbox/alloy/pull/83/commits/a885fbb9f131812540554ef12f9048d3a194b4bd)